### DISCUSSION TITLE
local compose up fail

### DISCUSSION THREAD

**Discussion by Physics-Morris** [alshedivat/al-folio#2812] [Q&A]
please help me. when I ran the command 
`docker compose up`
on a new laptop, I got the following error message

```
Attaching to jekyll-1
jekyll-1  | Entry point script running
jekyll-1  | bundler: failed to load command: jekyll (/usr/local/bundle/bin/jekyll)
jekyll-1  | /usr/local/bundle/gems/bundler-2.5.22/lib/bundler/resolver.rb:354:in `raise_not_found!': Could not find gem 'jekyll-diagrams' in locally installed gems. (Bundler::GemNotFound)
```

What should I do?

**Comment by george-gca** [comment]
Duplicate of #2747. Check our [FAQ](https://github.com/alshedivat/al-folio/blob/47ed26fb4ae3bdad22adcb3e23f31c40b9a90961/FAQ.md#i-am-trying-to-deploy-my-site-but-it-fails-with-could-not-find-gem-jekyll-diagrams-in-locally-installed-gems-how-do-i-fix-that).
